cardano-cli/src/Cardano/CLI/Parsers.hs

```diff
@@ -1,242 +1,35 @@
-        [ commandGroup "Key related commands"
-        , metavar "Key related commands"
-        , command' "keygen" "Generate a signing key."
-            $ Keygen
-                <$> parseCardanoEra
-                <*> parseNewSigningKeyFile "secret"
-                <*> parseFlag' GetPassword EmptyPassword
-                      "no-password"
-                      "Disable password protection."
-        , command'
-            "to-verification"
-            "Extract a verification key in its base64 form."
-            $ ToVerification
-                <$> parseCardanoEra
-                <*> parseSigningKeyFile
-                      "secret"
-                      "Signing key file to extract the verification part from."
-                <*> parseNewVerificationKeyFile "to"
-        , command'
-            "signing-key-public"
-            "Pretty-print a signing key's verification key (not a secret)."
-            $ PrettySigningKeyPublic
-                <$> parseCardanoEra
-                <*> parseSigningKeyFile
-                      "secret"
-                      "Signing key to pretty-print."
-        , command'
-            "signing-key-address"
-            "Print address of a signing key."
-            $ PrintSigningKeyAddress
-                <$> parseCardanoEra
-                <*> parseNetworkMagic
-                <*> parseSigningKeyFile
-                      "secret"
-                      "Signing key, whose address is to be printed."
-        , command'
-            "migrate-delegate-key-from"
-            "Migrate a delegate key from an older version."
-            $ MigrateDelegateKeyFrom
-                <$> parseCardanoEra -- Old CardanoEra
-                <*> parseSigningKeyFile "from" "Signing key file to migrate."
-                <*> parseCardanoEra -- New CardanoEra
-                <*> parseNewSigningKeyFile "to"
-        ]
-parseLocalNodeQueryValues :: Parser ClientCommand
-parseLocalNodeQueryValues =
-  subparser $ mconcat
-        [ commandGroup "Local node related commands"
-        , metavar "Local node related commands"
-        , command' "get-tip" "Get the tip of your local node's blockchain"
-            $ GetLocalNodeTip
-                <$> (ConfigYamlFilePath <$> parseConfigFile)
-                <*> optional (parseSocketPath "Socket of target node")
-        ]
-
-parseMiscellaneous :: Parser ClientCommand
-parseMiscellaneous = subparser $ mconcat
-  [ commandGroup "Miscellaneous commands"
-  , metavar "Miscellaneous commands"
-  , command'
-      "validate-cbor"
-      "Validate a CBOR blockchain object."
-      $ ValidateCBOR
-          <$> parseCBORObject
-          <*> parseFilePath "filepath" "Filepath of CBOR file."
-  , command'
-      "version"
-      "Show cardano-cli version"
-      $ pure DisplayVersion
-  , command'
-      "pretty-print-cbor"
-      "Pretty print a CBOR file."
-      $ PrettyPrintCBOR
-          <$> parseFilePath "filepath" "Filepath of CBOR file."
-  ]
-
+-}
```

Comment:
?? Is that some code that can just be dropped?

Reply:
It gets re-enabled in the next patch.